assets/js/main.js

```diff
@@ -9,3 +60,111 @@ openBtn.addEventListener('click', () => {
+                ${element.description}
+              </p>
+              <ul class="tag tag-feature mg-1 ff-inter flex">
+                <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">  ${element.tags[0]}  </a></li>
+                <li><a href="https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics">  ${element.tags[1]}  </a></li>
+                <li><a href="https://getbootstrap.com/">  ${element.tags[2]}  </a></li>
+                <li><a href="https://www.ruby-lang.org/en/"> ${element.tags[3]}  </a></li>
+              </ul>
+              <button class="btn open-btn"  data-id=${element.id} type="button">See Project</button>
+            </div>
+        </div>
+        <div class = "modal"> 
+          <div class = "modal-header flex">
+            <h3 class="mg-1 ff-crete">${element.header}</h3>
+            <img id="close-btn" src="./assets/img/close-icon.png" alt="close-icon">
+          </div>
+
+          <ul class="mg-2 mg-inline flex tag tag-featured">
+            <li><a href="https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics"> ${element.tags[0]} </a></li>
+            <li><a href="https://getbootstrap.com/">  ${element.tags[1]} </a></li>
+            <li><a href="https://www.ruby-lang.org/en/">  ${element.tags[2]} </a></li>
+          </ul>
+          <div class="flex modal-content">
+            <img class="mg-2" src="${element.img_dir}" alt="modal-image">
+            <div>
+              <p class="ff-inter">${element.content}</p>
+              <div class = " buttons mg-2 flex">
+                <button class="btn flex" type="button">See Live <img src="./assets/img/see-live-icon.png" alt="see-live-icon"> </button>
+                <button class="btn flex" type="button">See Source <img src="./assets/img/github.png" alt="github"> </button>
+              </div>
+            </div>
+          </div>
+        </div>
+        `;
+    } else {
+      (
+        container.innerHTML += `
+        <div class="mg-2 text-light project">
+          <h2 class="mg-inline bg-text">${element.header}</h2>
+          <p class="mg-1 mg-inline bg-text">
+                ${element.description}
+          </p>
+          <ul class="mg-1 mg-inline flex tag tag-project">
+            <li><a href="https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics"> ${element.tags[0]} </a></li>
+            <li><a href="https://getbootstrap.com/">  ${element.tags[1]} </a></li>
+            <li><a href="https://www.ruby-lang.org/en/">  ${element.tags[2]} </a></li>
+          </ul>
+          <button class="btn btn-project open-btn" data-id=${element.id} type="button">See Project</button>
+        </div>
+        <div class = "modal"> 
+          <div class = "modal-header flex">
+            <h3 class="ff-crete">${element.header}</h3>
+            <img id="close-btn" src="./assets/img/close-icon.png" alt="close-icon">
+          </div>
```

Comment:
Kindly fix the alignment of items on the popup so that your design looks as close to the [FIGMA design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)as possible.

- The image is too large in terms of `width` and takes too much space, leaving little space for the description text. Kindly make sure that you use about 60% of the container width for the image so that there's more room for the description text.
- Kindly fix the width and height of the buttons so that they are the same height and width as in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
- kindly fix the gap on the left margin so the all the popup content has equal left and right margins as in the Figma template.

![image](https://user-images.githubusercontent.com/38088818/169273893-39534cc9-f40a-4c3b-9eee-652c52d5f39e.png)


Figma:
![image](https://user-images.githubusercontent.com/38088818/169270990-4b6ca9c5-2d52-4bdd-ae39-ae452476eddd.png)